force-app/main/default/lwc/attendance/attendance.js

```diff
@@ -391,4 +414,39 @@ export default class Attendance extends NavigationMixin(LightningElement) {
             });
         }
     }
+
+    handleSortOption(event) {
+        if (!event || !event.detail || !event.detail.value) {
+            return;
+        }
+
+        this.sortAttendanceBy = event.detail.value;
+        this.sortServiceDeliveries(this.sortAttendanceBy);
+    }
+
+    sortServiceDeliveries(sortBy) {
+        if (this.serviceDeliveries) {
+            if (
+                sortBy.toLowerCase() ===
+                CONTACT_FIRST_NAME_FIELD.fieldApiName.toLowerCase()
+            ) {
+                this.serviceDeliveries.sort((a, b) => {
+                    return getChildObjectByName(a, "Contact__r").FirstName >
```

Comment:
Instead of having the conditional to check the field name, since we know the values are field api names, could we just instead do something like getChildObjectByName(a, "Contact__r")[sortBy]?